### PR TITLE
test: e2e test case for tablespace 

### PR DESCRIPTION
**Pull request by litaocdl**
The last ticket is the e2e test case. other commit are cherry pick as it is needed by e2e.

### CODE REVIEW

**Comment by litaocdl** [comment]
rebase and rerun a case

File: tests/e2e/tablespaces_test.go [hunk 1]
```diff
@@ -116,9 +158,91 @@ var _ = Describe("Tablespaces tests", Label(tests.LabelSmoke, tests.LabelBasic),
...
+				testUtils.ExecuteBackup(namespace, clusterBackupManifest, false, testTimeouts[testUtils.BackupIsReady], env)
+				AssertBackupConditionInClusterStatus(namespace, clusterName)
+			})
+
+			By("verifying the number of tars in minio", func() {
+				backupTars := minioPath(clusterName, "*.tar")
+				// as there are 2 tablespaces, we should have 3 tar files (data.tar + one tar per tablespace)
+				Eventually(func() (int, error) {
+					return testUtils.CountFilesOnMinio(namespace, minioClientName, backupTars)
+				}, 60).Should(BeEquivalentTo(3))
```
Comment (jsilvela) [inline tests/e2e/tablespaces_test.go:226]: This is surprising to me. At this point we should have 3 tablespaces, and therefore 4 tar files, not 3.

**Review by jsilvela — APPROVED** [review]

File: tests/e2e/tablespaces_test.go (hunk 1 above)
Comment (litaocdl) [reply tests/e2e/tablespaces_test.go]: yes, it should be 4

**Review by mnencia — APPROVED** [review]

**Pull request merged**